configs/15.0/packages/glibc/stage_2

```diff
@@ -118,7 +118,19 @@ atcfg_configure() {
 
 # Make build command
 atcfg_make() {
-	PATH=${at_dest}/bin:${PATH} ${SUB_MAKE}
+	if [[ ("${distro_fm}" == "ubuntu") || ("${distro_fm}" == "debian") ]]; then
+			if [[ "${build_arch}" == "ppc64le" ]]; then
+					user_defined_dir='user-defined-trusted-dirs=/lib/powerpc64le-linux-gnu:/usr/lib/powerpc64le-linux-gnu'
+			elif [[ "${build_arch}" == "ppc64" ]]; then
+					user_defined_dir='user-defined-trusted-dirs=/lib/powerpc64-linux-gnu:/usr/lib/powerpc64le-linux-gnu'
+			else    
+					user_defined_dir='user-defined-trusted-dirs=/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu'
```

Comment:
I think this is wrong, because we don't build AT for x86_64.
Is the code ever reaching this part?